tasks/release.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env sh
+
+release() {
+  # Clean node_modules
+  rm -rf node_modules && rm -f yarn.lock
+
+  # Install dependencies.
+  yarn
+
+  # Bump version.
+  yarn version --no-git-tag-version --new-version ${1:-patch}
+
+  # Get the new version number.
+  local version=`grep -oP '(?<="version": ")[\d.]+' package.json`
```

Comment:
Apologies, this version actually only works with GNU grep. It is not compatible with the BSD grep that ships with macOS. Here's a version that's compatible with both:

```suggestion
  local version=`grep -m1 version package.json | cut -d '"' -f4`
```

Reply:
Yeah, I was trying that and it was breaking the release process. I'll try out the new suggestion.

Reply:
Working fine now.